net/developer-guide/manage-presentation/convert-presentation/convert-powerpoint/convert-powerpoint-to-video/_index.md

```diff
@@ -0,0 +1,229 @@
+
+```c#
+using (Presentation presentation = new Presentation())
+
+{
+    // add smile shape and animate it
+    IAutoShape smile = presentation.Slides[0].Shapes.AddAutoShape(ShapeType.SmileyFace, 110, 20, 500, 500);
+    IEffect effectIn = presentation.Slides[0].Timeline.MainSequence.AddEffect(smile, EffectType.Fly, EffectSubtype.TopLeft, EffectTriggerType.AfterPrevious);
+    IEffect effectOut = presentation.Slides[0].Timeline.MainSequence.AddEffect(smile, EffectType.Fly, EffectSubtype.BottomRight, EffectTriggerType.AfterPrevious);
+    effectIn.Timing.Duration = 2f;
+    effectOut.PresetClassType = EffectPresetClassType.Exit;
+
+    using (var animationsGenerator = new PresentationAnimationsGenerator(presentation))
+    {
+        animationsGenerator.NewAnimation += animationPlayer =>
+        {
+            Console.WriteLine($"Total animation duration: {animationPlayer.Duration}");
+            
+            animationPlayer.SetTimePosition(0); // initial animation state
+            Bitmap bitmap = animationPlayer.GetFrame(); // initial animation state bitmap
+
+            animationPlayer.SetTimePosition(animationPlayer.Duration); // final state of the animation
+            Bitmap lastBitmap = animationPlayer.GetFrame(); // last frame of the animation
+            lastBitmap.Save("last.png");
+        };
+    }
+}
+```
+
+To make all animations in a presentation play at once, the [PresentationPlayer](https://reference.aspose.com/slides/net/aspose.slides.export/presentationplayer/) class is used. This class  takes a [PresentationAnimationsGenerator](https://reference.aspose.com/slides/net/aspose.slides.export/presentationanimationsgenerator/) instance and FPS for effects in its constructor and then call the `FrameTick` event for all the animations to get them played:
+
+```c#
+using (Presentation presentation = new Presentation("animated.pptx"))
+{
+    using (var animationsGenerator = new PresentationAnimationsGenerator(presentation))
+    using (var player = new PresentationPlayer(animationsGenerator, 33))
+    {
+        player.FrameTick += (sender, args) =>
+        {
+            args.GetFrame().Save($"frame_{sender.FrameIndex}.png");
+        };
+        animationsGenerator.Run(presentation.Slides);
+    }
+}
+```
+
+Then the generated frames can be compiled to produce a video. See the [Convert PowerPoint to Video](https://docs.aspose.com/slides/net/convert-powerpoint-to-video/#convert-powerpoint-to-video) section.
+
+
+
```

Comment:
Add the list of currently support transitions and shape animations (I mentioned it in the email)? 

Not sure why we skip it (may be this is required under some considerations)